charts/langflow-ide/values.yaml

```diff
@@ -53,7 +53,7 @@ langflow:
       failureThreshold: 3
       periodSeconds: 10
       timeoutSeconds: 5
-      initialDelaySeconds: 5
+      initialDelaySeconds: 35
```

Comment:
I had to increase the delay time in my k8s cluster, not sure if this happen to more people.